client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
Are you sure it's `type_method`? Seems like it's looking for `identifier_something`. I don't know the rule this is enforcing, but I'd like this method name to be more descriptive than just "second". Are you sure a field name after the underscore won't be accepted?